_posts/2021-04-06-run-minitest-on-github-actions-with-parallel-jobs-using-build-matrix.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title:  "Run Minitest on Github Actions with parallel jobs using build matrix"
+date:   2021-04-06 08:00:00 +0100
+author: "Artur Trzop"
+categories: continuous_integration
+og_image: "/images/blog/posts/run-minitest-on-github-actions-with-parallel-jobs-using-build-matrix/github-minitest.jpeg"
+---
+
+How to run Ruby on Rails tests in Minitest on Github Actions? What to do if tests are slow? How to manage complex workflows? You can use Github Actions build matrices to divide Minitest files between jobs and run the test suite much faster.
+
+<img src="/images/blog/posts/run-minitest-on-github-actions-with-parallel-jobs-using-build-matrix/github-minitest.jpeg" style="width:300px;margin-left: 15px;float:right;" alt="Minitest, Ruby, Github, Github Actions" />
+
+If your Minitest tests are taking dozen minutes and you would like to save some time for your Ruby engineering team then you could use tests parallelization on your CI server.
```

Comment:
"dozen minutes" means literally "12 minutes".
You could say "dozens of minutes" if you mean to say multiples of 12-minute periods (similarly to how in Polish we would say "dziesiątki" or "kilkadziesiąt")